tests/test_main.py

```diff
@@ -316,14 +316,14 @@ def test_elves_concert(elves, songs):
     "dwarves,feast_output",
     [
         (
-            [
-                DwarfWarrior(nickname="Thiddeal", favourite_dish="French Fries", hummer_level=3),
-                DwarfWarrior(nickname="Dwarf", favourite_dish="Caesar Salad", hummer_level=3),
-            ],
-            (
-                "Thiddeal is eating French Fries\n"
-                "Dwarf is eating Caesar Salad\n"
-            )
+                [
+                    DwarfWarrior(nickname="Thiddeal", favourite_dish="French Fries", hummer_level=3),
+                    DwarfWarrior(nickname="Dwarf", favourite_dish="Caesar Salad", hummer_level=3),
+                ],
+                (
+                        "Thiddeal is eating French Fries\n"
+                        "Dwarf is eating Caesar Salad\n"
+                )
         ),
         (
                 [
```

Comment:
Revert any changes done to the `tests/test_main.py` file. You could copy the content from the base repository, paste it to the current file, and commit changes.

Reply:
Well, actually, that what I did before :) 
I went to mate-academy / py-elves-and-dwarves -> Tests -> test_main.py and copied the content out of there and pasted to my test/main.py.
I will do it one more time. If I misunderstood, please inform me.